src/commands/operation.rs

```diff
@@ -71,27 +71,23 @@ fn cmd_op_log(
                 metadata.hostname
             )?;
             formatter.write_str(" ")?;
-            formatter.with_label("time", |formatter| {
-                formatter.write_str(
-                    &(if self.relative_timestamps {
-                        let mut f = timeago::Formatter::new();
-                        f.min_unit(timeago::TimeUnit::Microseconds).ago("");
-                        let mut duration =
-                            format_duration(&metadata.start_time, &metadata.end_time, &f);
-                        if duration == "now" {
-                            duration = "less than a microsecond".to_string()
-                        }
-                        let start = format_timestamp_relative_to_now(&metadata.start_time);
-                        format!("{start}, lasted {duration}",)
-                    } else {
-                        format!(
-                            "{} - {}",
-                            format_absolute_timestamp(&metadata.start_time),
-                            format_absolute_timestamp(&metadata.end_time)
-                        )
-                    }),
-                )
-            })?;
+            if self.relative_timestamps {
+                let mut f = timeago::Formatter::new();
+                f.min_unit(timeago::TimeUnit::Microseconds).ago("");
+                let mut duration = format_duration(&metadata.start_time, &metadata.end_time, &f);
+                if duration == "now" {
+                    duration = "less than a microsecond".to_string()
```

Comment:
It would then say "5 min ago, lasted now". The code does get triggered in tests.

I guess you're right that it's hard to imagine a world where this would trigger in real life. But we can always hope.

Reply:
It does happen a few times in my repo (e.g. the initial operation, but I think that explicitly sets the same timestamp for both start and end)